salt/modules/rpm.py

```diff
@@ -422,6 +422,14 @@ def info(*packages, **attr):
         salt '*' lowpkg.info apache2 bash attr=version
         salt '*' lowpkg.info apache2 bash attr=version,build_date_iso,size
     '''
+    # LONGSIZE is not a valid tag for all versions of rpm. If LONGSIZE isn't
+    # available, then we can just use SIZE for older versions. See Issue #31366.
+    rpm_tags = __salt__['cmd.run_all']('rpm --querytags')
+    rpm_tags = rpm_tags.get('stdout').split('\n')
```

Comment:
Since we're not paying attention to the retcode or other information that is returned in ``cmd.run_all``, the above two lines can be more concisely written as follows:

```python
rpm_tags = __salt__['cmd.run_stdout'](
    ['rpm', '--querytags'],
    python_shell=False).splitlines()
```